platform-common-date/src/main/java/com/softicar/platform/common/date/CommonDateI18n.java

```diff
@@ -18,6 +18,14 @@
 		.de("%sh %smin %ss");
 	I18n2 ARG1MIN_ARG2S = new I18n2("%smin %ss")//
 		.de("%smin %ss");
+	I18n1 ILLEGAL_DATE_SPECIFICATION_ARG1 = new I18n1("Illegal date specification: %s")//
```

Comment:
* `date format` would not be correct, since 2021-11-31 has a valid format but is still an illegal specification, specification is not meant here as a format specification but rather as a value specification.
 * `date input` implies that it was given as an input as in `Datumseingabe`, while the word that I want is `Datumsangabe`, while is most cases here it would probably be correct, although DayParsed can also be used without and input, e.g. some date received over the network

Reply:
I am undecided here.

Reply:
I see. So, how about plain and simple `Illegal date: %s`?
`-angabe`/`-eingabe` does not seem to add valuable information.

Reply:
I tried it but it somehow leads to ambiguous or misleading texts. I would like to keep the name as it is.